### PR TITLE
Bump Graphiql to fix layout

### PR DESCRIPTION
**Pull request by Kowjako**
Fix for issue https://github.com/graphql-dotnet/server/issues/1129

### CODE REVIEW

File: src/Ui.GraphiQL/Internal/graphiql.cshtml [hunk 1]
```diff
@@ -52,8 +52,7 @@
    -->
   <link
     rel="stylesheet"
-    href="https://unpkg.com/graphiql@1.5.1/graphiql.min.css"
-    integrity="sha384-1YHEU+Xy8hlKYAZ26WTz+JQEPMM6i/Mx5m8umMkSZChlzSYmq7RqyCyRbGqrILVZ"
+    href="https://unpkg.com/graphiql/graphiql.min.css"
```
Comment (Shane32) [inline src/Ui.GraphiQL/Internal/graphiql.cshtml:55]: I think we should maintain a specific version tag and integrity hash.

File: src/Ui.GraphiQL/Internal/graphiql.cshtml [hunk 2]
```diff
@@ -52,8 +52,7 @@
...
     crossorigin="anonymous"
   />
   <script
```
Comment (Shane32) [inline src/Ui.GraphiQL/Internal/graphiql.cshtml:58]: On the next line there is a reference to graphiqlWithExtensions. Is this old version compatible with the new version of GraphiQL ? All other features still work?

File: src/Ui.GraphiQL/Internal/graphiql.cshtml (hunk 1 above)
Comment (Kowjako) [reply src/Ui.GraphiQL/Internal/graphiql.cshtml]: Sure, updated

File: src/Ui.GraphiQL/Internal/graphiql.cshtml (hunk 2 above)
Comment (Kowjako) [reply src/Ui.GraphiQL/Internal/graphiql.cshtml:59]: Yes, ive checked and it works as expected

**Comment by Kowjako** [comment]
Btw new Graphiql is awesome 
image

**Review by Shane32 — APPROVED** [review]

**Comment by ame0410** [comment]
@Kowjako will this PR enable the new GraphiQL 2.0 look and feel?
Also, will this be released in GraphQL.Server.Ui.GraphiQL 8.x?

**Comment by Shane32** [comment]
Yes, this PR has been merged for 8.x.  It will bump GraphiQL to 3.2.0

**Comment by ame0410** [comment]
Thanks, @Shane32 . We will be eagerly waiting for 8.x :)

**Pull request merged**